rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
```

Comment:
- [ ] For your `add_rental` method the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association, it may look something like this :point_down: 

 ```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```